certbot/docs/using.rst

```diff
@@ -529,6 +529,70 @@ revocation from any ACME account::
 
   certbot revoke --cert-path /etc/letsencrypt/live/example.com/cert.pem --key-path /etc/letsencrypt/live/example.com/privkey.pem
 
+.. _deleting:
+
+Deleting certificates
+---------------------
+
+If you need to delete a certificate, use the ``delete`` subcommand.
+
+.. note:: Read this and the `Safely deleting certificates`_ sections carefully. This is an irreversible operation and must
+          be done with care.
+
+.. note:: Do not manually delete certificate files from inside ``/etc/letsencrypt/``. Always use the ``delete`` subcommand.
+
+A certificate may be deleted by providing its name with ``--cert-name``. \
+You may find its name using ``certbot certificates``.
+
+Otherwise, you will be prompted to choose one or more
+certificates to delete::
+
+  certbot delete --cert-name example.com
+  # or to choose from a list:
+  certbot delete
+
+Safely deleting certificates
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
```

Comment:
Loving the steps version! But now taking a step back, it could be more explicit that this top matter is a description of what is to come, rather than a prerequisite before taking the steps.

Drafted a version of what it could look like to clear that up:

```
Deleting a certificate without following the proper steps can result in a non-functioning server. To safely delete a certificate, follow all the steps below to make sure that references to a certificate are removed from the configuration of any installed server software (Apache, nginx, Postfix, etc) *before* deleting the certificate.

To explain further, when installing a certificate, Certbot modifies Apache or nginx's configuration to load the certificate and its
private key from the ``/etc/letsencrypt/live/`` directory. Before deleting a certificate, it is necessary to undo that modification, by removing any references to the certificate
from the webserver's configuration files.

Follow these steps to safely delete a certificate:```

Reply:
Good point. I don't love that the text is repeating itself a little bit, but it's a small price to pay for clarity :+1: .